test/branch.ts

```diff
@@ -0,0 +1,198 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {describe, it, before} from 'mocha';
+import {assert, expect} from 'chai';
+import {logger, octokit, setup} from './util';
+import * as sinon from 'sinon';
+import {
+  branch,
+  getBranchHead,
+  createRef,
+} from '../src/github-handler/branch-handler';
+
+before(() => {
+  setup();
+});
+
+describe('Branch module', async () => {
+  const origin = {owner: 'octocat', repo: 'HelloWorld'};
+  const branchName = 'test-branch';
+  const branchResponseBody = await import(
+    './fixtures/create-branch-response.json'
+  );
+  const branchResponse = {
+    headers: {},
+    status: 200,
+    url: 'http://fake-url.com',
+    data: branchResponseBody,
+  };
+
+  describe('Octokit list branch function', () => {
+    it('is passed the correct parameters', async () => {
+      // setup
+      const getBranchStub = sinon
+        .stub(octokit.repos, 'getBranch')
+        .resolves(branchResponse);
+      // // tests
+      await getBranchHead(logger, octokit, origin, 'master');
```

Comment:
We should still capture the return value and assert its value is correctly parsed.